go-controller/pkg/ovn/management-port.go

```diff
@@ -78,36 +63,30 @@ func configureManagementPortWindows(clusterSubnet, clusterServicesSubnet,
 		return fmt.Errorf("Failed to parse clusterSubnet %v : %v", clusterSubnet, err)
 	}
 	// Checking if the route already exists, in which case it will not be created again
-	stdoutStderr, err := exec.Command("route", "print", "-4", fmt.Sprintf("%s", clusterIP)).CombinedOutput()
+	stdout, stderr, err := util.RunRoute("print", "-4", clusterIP.String())
 	if err != nil {
-		logrus.Debugf("Failed to run route print, stderr: %q, error: %v", stdoutStderr, err)
+		logrus.Debugf("Failed to run route print, stderr: %q, error: %v", stderr, err)
 	}
 
 	var interfaceIndex string
-	if strings.Contains(fmt.Sprintf("%s", stdoutStderr), fmt.Sprintf("%s", clusterIP)) {
+	if strings.Contains(stdout, clusterIP.String()) {
 		logrus.Debugf("Route was found, skipping route add")
 	} else {
-		args = []string{"$(Get-NetAdapter", "|", "Where",
-			"{", "$_.Name", "-Match", fmt.Sprintf("\"%s\"", interfaceName), "}).ifIndex"}
-		logrus.Debugf("Executing 'powershell %s'", strings.Join(args, " "))
-
-		stdoutStderr, err := exec.Command("powershell", args...).CombinedOutput()
+		stdout, stderr, err = util.RunPowershell("$(Get-NetAdapter", "|", "Where",
+			"{", "$_.Name", "-Match", fmt.Sprintf("\"%s\"", interfaceName), "}).ifIndex")
 		if err != nil {
-			logrus.Errorf("Failed to fetch interface index, stderr: %q, error: %v", stdoutStderr, err)
+			logrus.Errorf("Failed to fetch interface index, stderr: %q, error: %v", stderr, err)
 			return err
 		}
-		interfaceIndex = strings.TrimSpace(fmt.Sprintf("%s", stdoutStderr))
+		interfaceIndex = strings.TrimSpace(stdout)
```

Comment:
Minor nit, I don't think ``strings.TrimSpace`` is needed here anymore since it's being already done inside ``util.RunPowershell`` method. You could just replace the stdout from above and put interfaceIndex.